utils/check_tests_in_ci.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
+# Copyright 2025-present, the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Check that all tests are called in CI."""
```

Comment:
Note: another way to be sure of that would simply be to run `pytest tests` in the CI instead of calling each one individually. It doesn't look as good in the github report though

Reply:
Another advantage of running `pytest tests` altogether is that if test B fails, then test C will still be triggered. It's not the case when calling tests iteratively.

Reply:
I also agree: why not just calling `pytest tests`?